compiler/resolution/functionResolution.cpp

```diff
@@ -10133,11 +10156,33 @@ static void lowerRuntimeTypeInit(CallExpr* call,
       runtimeTypeToValueCall->insertAtTail(sub);
     }
   }
-
+ 
   // Add the argument indicating if this is a noinit
   Symbol* isNoInit = noinit ? gTrue : gFalse;
   runtimeTypeToValueCall->insertAtTail(isNoInit);
 
+  // Add the argument indicating if this is defined as const
+  bool insideCoerceCopyOrMove = false;
```

Comment:
I don't think we need this workaround here - the module code can just call the function this lowers to.